documentation/tutorial.md

```diff
@@ -1,47 +1,54 @@
-# Charmed MongoDB tutorial:
-Ready to try out Charmed MongoDB? This tutorial will guide you through the steps to get Charmed MongoDB up and running. 
+# Charmed MongoDB tutorial
+The Charmed MongoDB Operator delivers automated operations management from [day 0 to day 2](https://codilime.com/blog/day-0-day-1-day-2-the-software-lifecycle-in-the-cloud-age/) on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production-ready data platform [on top of Juju](https://juju.is/). As a first step this tutorial shows you how to get Charmed MongoDB up and running, but the tutorial does not stop there. Through this tutorial you will learn a variety of operations, everything from adding replicas to advanced operations such as enabling Transcript Layer Security (TLS). In this tutorial we will walk through how to:
+- Set up your environment using LXD and Juju.
+- Deploy MongoDB using a single command.
+- Access the admin database directly.
+- Add high availability with replication.
+- Change the admin password.
+- Automatically create MongoDB users via Juju relations. 
+- Enable secure transactions with TLS.
 
-## Minimum requirements:
-Before we start be sure you have the following requirements:
-- Ubuntu 20.04(focal) or later.
+While this tutorial is aimed to guide and teach you as you deploy Charmed MongoDB. It will be most beneficial if you already have a familairy with: 
```

Comment:
```suggestion
While this tutorial intends to guide and teach you as you deploy Charmed MongoDB, it will be most beneficial if you already have a familiarity with: 
```